arkouda/pdarrayIO.py

```diff
@@ -581,3 +588,146 @@ def save_all(columns : Union[Mapping[str,pdarray],List[pdarray]], prefix_path :
             first_iter = False
         else:
             arr.save(prefix_path=prefix_path, dataset=name, file_format=file_format, mode='append')
+
+
+def import_data(read_path: str, write_file: str = None,
+                return_obj: bool = False, index: bool = False):
+    """
+    Import data from Pandas to Arkouda readable format
+
+    Parameters
+    __________
+    read_path: str
+        path to file where pandas data is stored. This can be glob expression for parquet formats.
+    write_to: str, optional
+        path to file to write arkouda formatted data to. Only write file if provided
+    return_obj: bool, optional
+        Default False. When True return the Arkouda DataFrame object
+    index: bool, optional
+        Default False. When True, maintain the indexes loaded from the pandas file
+
+    Raises
+    ______
+    RuntimeWarning
+        - Export attempted on Parquet file. Arkouda formatted Parquet files are readable by pandas.
```

Comment:
I'm confused by the warning. Are we importing or exporting? What is an "Arkouda formatted parquet file"?

Reply:
This is actually not valid anymore. I had something working differently that did not account for multi-locale files and neglected to remove the leftover warning. It has been removed.